src/js/nav/globalNav.js

```diff
@@ -17,7 +42,21 @@ export const options = Object.freeze([{
     title: 'Remediations'
 }, {
     id: 'cost-management',
-    title: 'Cost Management'
+    title: 'Cost Management',
+    subItems: [
+        {
+            id: '',
```

Comment:
Should we add IDs here?

Reply:
I guess it doesn't really matter since `'' == undefined == false` but I like to indicate that `overview` leads to nowhere